src/app/common/stores/Overview.store.ts

```diff
@@ -27,7 +27,7 @@ class OverviewStore extends BaseStore {
       return;
     }
     new ApiRequest({
-      url: `${config.links.API_BASE_URL}/api/currency/convert?eth=${totalEth}`,
+      url: `${config.links.API_BASE_URL}/currency/convert?eth=${totalEth}`,
```

Comment:
This api have two path parameters: https://api.stage.ssv.network/documentation/#/Finance/FinanceController_currencyConvert

Reply:
done